search/intermediate_result.cpp

```diff
@@ -76,33 +108,35 @@ void ProcessMetadata(FeatureType const & ft, Result::Metadata & meta)
   meta.m_isInitialized = true;
 }
 
-PreResult1::PreResult1(FeatureID const & fID, PreRankingInfo const & info) : m_id(fID), m_info(info)
+PreRankerResult::PreRankerResult(FeatureID const & fID, PreRankingInfo const & info)
```

Comment:
s/fID/fid/

Reply:
Or, even, "id".

Reply:
I was thinking to rename it to featureId, actually.
id for now
done